### DISCUSSION TITLE
Adobe Commerce 2.4 vs OpenMage Performance

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#2935] [General]
Maybe some of you have read this article since the end of 2020.

https://medium.com/swlh/adobe-magento-2-4-vs-openmage-magento-1-9-open-source-performance-2021-c710406d3dc9

Unfortunately, the performance leaves much to be desired in the M2. More and more people are disappointed that major changes are needed to solve this issue.

Personally, I had no problem adapting myself to M2, but those whose stores I manage did not want to migrate. They told me that the Backend interface is ugly, that it doesn't look like M1 at all. Obviously, the migration, the adaptation of the extensions, brings good money, but I think that many technical aspects have been lost sight of and the wishes of the clients have not been taken into account. I know cases when they were moved with stories about vulnerability, about attacks, malfunctions. I am sorry to say it in public but this was marketing bullshit.

After almost 3 years since the EOL established by Magento and the abandonment of the platform much earlier due to the priority they gave to the M2 launch, I can say that it was a mistake. Let's not forget that we didn't have official PHP 7 support for a long time and it was offered by Inchoo through an extension. They broke something that was good and a whole guild of talented developers, who created important extensions, had to abandon customers.

I think that OpenMage still has a chance to remain in the top choices for anyone who wants a high-performance, adaptable, easy-to-use shopping cart. too bad we don't have the financial strength of a corporation because we would have been far away now.

I wrote this post to motivate all those who contribute to this project today that their work is not in vain. Thank you.

**Comment by fballiano** [comment]
magento2 was one of the biggest mistakes in tech ever. they destroyed the absolute leadership position that magento1 had and fractured the most awesome tech community.

but, somebody made great money in the process anyway.
